vrk-kpa/suomidigi
# SUOPA-782 News

To test:
- `make fresh`
- Log in and go to create **few** News items https://suomidigi.docker.sh/node/add/news_item
- After you've managed to create a few news items, go to https://suomidigi.docker.sh/node/add/page
   - Give the page a title
   - Add a View paragraph with View: Sisältösivu: Uutiset and Näyttö Lista
   - Add URL alias to /tiedotteet/
   - Add the page to a menu to the position of your choice
   - Save the node
- Check that the page you've create has a similar design to https://xd.adobe.com/view/d3c60a1d-1d76-4a8b-4797-030e9c2e717f-a5f7/screen/b24cb7f9-6eab-442f-98b4-d0af8376cd80/Suomidigi-2020-tiedote-xl-1
- Click one of the news item and check that it has a similar design to https://xd.adobe.com/view/d3c60a1d-1d76-4a8b-4797-030e9c2e717f-a5f7/screen/a1456ab6-dbf6-4627-8380-b0cecde48d40/Suomidigi-2020-tiedote-xl
